health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/AppConfigurationParentLayer.js

```
@@ -128,19 +187,56 @@
   };
   return map[label] || label;
 }
+
+const getTypeAndFormat = (field) => {
+  switch (field.type) {
+    case "textInput":
+      return { type: "string", format: "string", maxLength: field?.maxLength };
+    case "textArea":
+      return { type: "string", format: "textArea", charCount: field?.maxLength };
+      break;
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)